.github/workflows/testing.yml

```diff
@@ -138,6 +149,10 @@ jobs:
             '~5.0.0',
             '~5.1.0',
             '~5.2.0',
+            '~5.3.0',
+            '~5.4.0',
+            '~5.5.0',
+            '~5.6.0',
```

Comment:
See [this Slack thread](https://pinecone-io.slack.com/archives/C05MYEX7T2L/p1736447673586879) for more color on this change